salt/etc-hosts/init.sls

```diff
@@ -14,6 +14,10 @@ dummy_step:
     - name: "echo saltstack bug 14553"
 {% endif %}
 
+{# Velum container will not see any updates of the /etc/hosts. It can't be fixed with bind-mount #}
+{# of /etc/hosts in the container, because of fileblock.replace copies the new file over the old /etc/hosts. #}
+{# So the old /etc/hosts will remain mounted in the container (as bind-mount works at inode level). #}
+{# For more info see https://github.com/kubic-project/salt/pull/265#issuecomment-337256898 #}
```

Comment:
MINOR: as long as nothing has to be rendered within the comment, you can just comment without the curly braces

Reply:
Yes, I agree. I just wanted to be consistent with other comments in this file. Thank you